package.json

```diff
@@ -100,7 +100,7 @@
     "yargs": "^6.3.0"
   },
   "devEngines": {
-    "node": "8.x || 9.x"
+    "node": "8.x"
```

Comment:
Please don't forget to change this back :-)

Reply:
Yes, definitely, thanks!